### ISSUE TITLE
Missing sqlite subdirecotory in melpa

### ISSUE THREAD

**Issue by flub** [magit/emacsql#111]
When using magit/forge after upgrading to melpa version emacsql-20230222.1938 the error `No EmacSQL SQLite binary available, aborting` is printed in modeline and messages buffer.  And magit/forge can no longer create pull requests etc.

This message comes from `(emacsql-sqlite-compile 2)` which looks for the `sqlite/Makefile` file in the installed package.  Unfortunately it seems melpa is not including the `sqlite/` directory, copying this directory into the local package directory makes everything work flawless.

This can be reproduced by installing emacsql from melpa, version 20230222.1938 and then trying to use it somewhere, like for magit/forge.

I apologise if this is the wrong place to report this, this could be a melpa issue instead.

**Comment by tarsius** [comment]
I've just updated the Melpa recipe to include that directory. It can take a few hours for an update to appear there and I am not 100% sure this change will trigger the necessary rebuild.

> I apologise if this is the wrong place to report this, this could be a melpa issue instead.

I am just glad I got a failry early report; doesn't matter where. It's a bit complicated, see https://github.com/melpa/melpa/commit/4872ef038dbbf67008bfa7951574ee372d6ff68d. :scream:
